lstchain/tools/lstchain_create_irf_files.py

```diff
@@ -259,25 +310,59 @@ def start(self):
 
         gammas = self.mc_particle["gamma"]["events"]
 
-        self.log.info(f"Using fixed G/H cut of {self.fixed_cuts.fixed_gh_cut}")
-
-        gammas = self.event_sel.filter_cut(gammas)
-        gammas = self.fixed_cuts.allowed_tels_filter(gammas)
-        gammas = self.fixed_cuts.gh_cut(gammas)
-
-        if self.point_like:
-            gammas = self.fixed_cuts.theta_cut(gammas)
-            self.log.info('Theta cuts applied for point like IRF')
-
         # Binning of parameters used in IRFs
         true_energy_bins = self.data_bin.true_energy_bins()
         reco_energy_bins = self.data_bin.reco_energy_bins()
         migration_bins = self.data_bin.energy_migration_bins()
         source_offset_bins = self.data_bin.source_offset_bins()
 
+        gammas = self.event_sel.filter_cut(gammas)
+        gammas = self.cuts.allowed_tels_filter(gammas)
+
+        if self.energy_dependent_gh:
+            self.gh_cuts_gamma = self.cuts.energy_dependent_gh_cuts(
+                gammas, reco_energy_bins, min_value=0.1, max_value=0.95
+            )
+            gammas = self.cuts.apply_energy_dependent_gh_cuts(
+                gammas, self.gh_cuts_gamma
+            )
+            self.log.info(
+                f"Using gamma efficiency of {self.cuts.gh_efficiency}"
```

Comment:
% missing in message

Reply:
Not if we adapt our comment of not changing the value.